installation/cmd/run-from-kyma.sh

```diff
@@ -17,4 +17,4 @@ mv -f ${OVERRIDES_MINIKUBE_PATH}.bak $OVERRIDES_MINIKUBE_PATH
 
 # TODO: Remove it after next CLI release
 echo "Adding Compass entries to /etc/hosts...\n"
```

Comment:
Could you please remove `\n`? I've just noticed this mistake, the `\n` is printed in the console. I know it's not your fault, but we would be grateful for the change 😄 Thanks!